tests/integration/main.go

```diff
@@ -96,6 +100,22 @@ func (tr TestRun) runStep(step Step, verbose bool) {
 	pretty.Print(actualState)
```

Comment:
I won't block merging on this, but it might be time to take this out or do something more intelligent with it. Right now I believe the steps are being interleaved on the console which makes printing it useless

Reply:
Yea I wonder if a good way to print concurrent tests even exists 🤔

Reply:
No, don't worry about it. There isn't actually a good reason to print it at all since it will just match the steps exactly, and if it doesn't, it will throw an error. I just had this in there originally for smoke testing